src/crate/theme/rtd/crate/footer.html

```diff
@@ -85,3 +85,7 @@
     </div>
   </div>
 </footer>
+
+<!--esi
+<esi:include src="https://crate.io/navi-newsletter.php" />
```

Comment:
When navigating to https://crate.io/navi-newsletter.php, the JavaScript console shows
```javascript
Uncaught ReferenceError: MktoForms2 is not defined
```

Reply:
is your js disabled again? ;)

Reply:
Oh right, after unblocking this in the NoScript extension, everything works flawlessly. Sorry for the noise!

Reply:
np, did you see the fallback email adress?